_posts/en/newsletters/2018-07-10-newsletter.md

```diff
@@ -204,10 +204,65 @@ to pursue later." ([source][pwuille comment])
   Bitcoin businesses both internally and hosted on behalf of their
   customers, so some readers of this newsletter may wish to investigate.
 
-- **Building on Bitcoin:** last week was a Bitcoin technology conference
-  well attended by Bitcoin protocol developers.  A [video][bob video]
-  is available, as are several [transcripts][bob transcripts] by Bitcoin
-  developer Bryan Bishop (kanzure).
+## Building on Bitcoin
 
+[**Building on Bitcoin**][bob website] was a Bitcoin technology conference that took
+place in Lisbon last week. It was well attended by both Bitcoin protocol
+developers and applications engineers.  A [video][bob video]
+is available, as are several [transcripts][bob transcripts] by Bitcoin
+developer Bryan Bishop (kanzure).
+
+[bob website]: https://building-on-bitcoin.com/
 [bob video]: https://www.youtube.com/watch?v=XORDEX-RrAI
 [bob transcripts]: http://diyhpl.us/wiki/transcripts/building-on-bitcoin/2018/
+
+The following talks may be of particular interest to Bitcoin Optech
+companies:
+
+- [**Merchant adoption**][bitrefill video] - [Sergej Kotliar][sergej], CEO of
+  Bitrefill gave a personal account of the fee market spike at the end of last
+  year, important UX considerations for Bitcoin and and lightning payments, and
```

Comment:
"and and" typo

Also there are several spellings of Lightning with a lower case l.

Reply:
FYI I attempted to make these changes myself in the GitHub editor, but I get a 404 when trying to create a PR...